vizro-core/src/vizro/__init__.py

```diff
@@ -1,16 +1,19 @@
+import json
 import logging
 import os
+from pathlib import Path
 
 import plotly.io as pio
 from dash.development.base_component import ComponentRegistry
 
 from ._constants import VIZRO_ASSETS_PATH
-from ._themes import dark, light
 from ._vizro import Vizro, _make_resource_spec
 
 logging.basicConfig(level=os.getenv("VIZRO_LOG_LEVEL", "WARNING"))
-pio.templates["vizro_dark"] = dark
-pio.templates["vizro_light"] = light
+
+base_path = Path(__file__).resolve().parent / "_themes"
```

Comment:
```suggestion
base_path = Path(__file__).parent / "_themes"
```
I don't think `resolve` should be needed here.